requirements.tests.txt

```diff
@@ -19,7 +19,8 @@ eventlet==0.30.2  # pyup: <0.30.3
 gevent==20.9.0
 # the bottom pin is for limbo test runs, as latest version doesn't work with
 # newer celery versions
-redis>=2.10.6,<=3.2.1
+redis>=3.4.1;python_version > "3.5"
+redis>=2.10.6,<=3.2.1;python_version <= "3.5"
```

Comment:
It looks like we could be good with replacing both of the above with:
```
redis>=2.10.6
```
py35 passes with the latest `redis` version.

And the comment above could be updated as well I suppose?

Reply:
Yes, but the test for minimal version won`t as there an older version of kombu is allowed, so this change is not an options without bumping minimal requirements